libtransmission/torrent-files.cc

```diff
@@ -361,15 +366,17 @@ namespace
 {
     switch (ch)
     {
+    case ':':
+#if !defined(__APPLE__) // only ':' is reserved on APFS
     case '"':
     case '*':
     case '/':
```

Comment:
I'm confident that this one is reserved.

Reply:
I've updated the PR